src/main/java/com/kodilla/ecommercee/controller/UserController.java

```diff
@@ -37,11 +37,8 @@ public void createUser(@RequestBody UserDto userDto) {
     }
 
     @PutMapping("blockUser")
-    public void blockUser(@RequestParam Long userId) throws UserNotFoundException {
-        UserDto userToBlockDto = userMapper.mapToUserDto(userService.getUser(userId).orElseThrow(UserNotFoundException::new));
-        userToBlockDto.setIsBlocked(true);
-        User userToBlock = userMapper.mapToUser(userToBlockDto);
-        userService.saveUser(userToBlock);
+    public User blockUser(@RequestParam Long userId) throws UserNotFoundException {
```

Comment:
Nie wiem czy to ma znaczenie skoro działa ale pomyśl o tym co pisał Andrzej 
"Potrzebne jest całe Dto? Nie wystarczy przekazać userId i na tej podstawie wyciągnąć w service resztę?"

Reply:
Hej, własnie wydaje mi się, że to zaimplementowałem, skoro metoda nie posiada teraz @RequestBody UserDto tylko na podstawie podania wyłącznie userId zmienia wyłącznie jego wartość isBlockec

Reply:
Dobra, zatwierdzimy to jak jest, żeby zmergować pullrequest, ewentualnie na spotkaniu zapytamy Andrzeja.